docs/developer-guide/releasing.md

```diff
@@ -125,26 +125,27 @@ checks to see if the release came out correctly:
 If something went wrong, damage should be limited. Depending on the steps that
 have been performed, you will need to manually clean up.
 
-* Delete release tag (i.e. `v1.6.0-rc2`) created on GitHub repository. This
-  will immediately set release (if created) to `draft` status, invisible for
+* Delete the release tag (e.g. `v1.6.0-rc2`) created in the GitHub repository. This
+  will immediately set the release (if created) to `draft` status, invisible to the
   general public.
-* Delete the draft release (if created) from `Releases` page on GitHub
+* Delete the draft release (if created) from the `Releases` page on GitHub
 * If Docker image has been pushed to DockerHub, delete it
-* If commits have been performed to the release branch, revert them. Paths that could have been commited to are:
+* If commits have been performed to the release branch, revert them. Paths that could have been committed to are:
     * `VERSION`
     * `manifests/*`
 
 ### Post-process manual steps
 
 For now, the only manual steps left are to
 
-* update stable tag in GitHub repository to point to new release (if appropriate)
+* update stable tag in the GitHub repository to point to new the release (if appropriate)
+* update the `VERSION` file on `master` if this is a new major release
```

Comment:
This bullet was missing from the manual steps which is the main reason I'm visiting this file.

The rest of the changes are to make app.grammarly.com / my sense of grammar happy.